packages/curve-ui-kit/src/shared/icons/FilterIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import { createSvgIcon } from '@mui/material/utils'
+
+export const FilterIcon = createSvgIcon(
+  <svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      fill-rule="evenodd"
```

Comment:
`fillRule` and `clipRule`. I know the linter will complain about this tsx casing

Reply:
Hmm the build looks OK though :cry:

Reply:
Huh weird, running the linter locally gives no warnings either. Pretty sure I had to fix the casing a while back 😅

Reply:
It gives warnings on dev mode but maybe something we can configure better later